third_party/openocd/BUILD

```diff
@@ -51,6 +51,7 @@ configure_make(
         "--enable-remote-bitbang",
     ],
     copts = ["-Wno-error=unused-variable"],
+    env = {"CCACHE": "none"},
```

Comment:
Neat! How did you figure out that it's looking for this env variable? I have a very slight preference for a configure flag if one exists.

Reply:
The logs showed OpenOCD running `jimctl`'s configure script, which which uses [this TCL file](https://github.com/msteveb/jimtcl/blob/36430b868343ff9315ab342ee531b31bce6b3c73/autosetup/cc.tcl#L15) to find `$CCACHE`/`ccache`.

Yeah a flag would be nicer, but I don't think there is one. It would have to pass from OpenOCD -> jimctl -> this TCL.

Reply:
Thanks for the details! Definitely LGTM.